critiquebrainz/frontend/forms/review.py

```diff
@@ -16,7 +16,15 @@ def __call__(self, form, field):
         if form.state.data == "draft":
             return
         l = len(field.data) if field.data else 0
-        if l < self.min or self.max != -1 and l > self.max:
+        if (l < self.min or self.max != -1 and l > self.max) and (l != 0):
```

Comment:
Can you explain what this change does? I even forgot how the original check works.